### DISCUSSION TITLE
Questions About Comparing Bitmagnet and BTDig, Installation on Raspberry Pi vs. Manjaro, Filtering Options, and Demo Version Suggestions

### DISCUSSION THREAD

**Discussion by 8ullyMaguire** [bitmagnet-io/bitmagnet#329] [Q&A] [answered]
Hello, I'm interested in self-hosting a torrent indexing service like BTDig. I'm considering installing bitmagnet, but I'm not very familiar with self-hosted services. Is there a comparison between these two available somewhere? What kind of filtering options does bitmagnet offer? For example, can you filter by file size, date added, number of seeders, etc?

I have a Raspberry Pi and a Manjaro Linux system. Would there be any benefits to installing it on the Pi, such as easier access from an Android phone, or would it be essentially the same on either system? Where can I learn more about using these self-hosted services effectively?

I've used BTDig to search for old files, and I'd like to have a similar index in bitmagnet. Is there an archived index as extensive as the one from BTDig that could be imported into bitmagnet?

I've installed bitmagnet-git from the AUR on Manjaro, as mentioned on https://bitmagnet.io/external-resources.html, but I can't find the app launcher. Do I need to use a systemctl command or something similar to start it?

I think it would be great to have an ad-supported demo version of bitmagnet. This could help with funding, increase visibility, and allow users to test the project before self-hosting, thereby improving discoverability. Perhaps some of the ad revenue could be used to set up issue bounties, encouraging more contributions to the project. Please consider creating a demo for the project; I would really appreciate it, especially if it includes a large database like BTDig or offers more filtering options than those commonly found in similar torrent search engines.

**Comment by mgdigital** [comment]
Hi, this project isn't affiliated with either Magnetico or BTDigg, and neither of those projects have been maintained for many years. There is no app launcher for Bitmagnet, you would access it via the web app assuming the http server is running. The Manjaro package is unofficial - if you're having trouble with it I'd suggest following the official installation instructions.

**Reply by 8ullyMaguire** [reply]
My bad; every time I wrote 'Magnetico' above, I was referring to 'bitmagnet'. How do the filtering options compare to BTDigg?

**Comment by mgdigital** [comment, marked as answer]
I'm not very familiar with BTDig. It seems not to have had a code update in 11 years so self hosting it might not be advisable. 

With Bitmagnet you can order by size, seeders etc but not filter. You can filter by a bunch of other criteria. Really the best way to learn more about Bitmagnet is to try it :)

There's no plan for an official public instance with unfiltered content from DHT, it would be too complicated legally. There might be a demo instance at some point but with public domain content only. Making money from ads is not a direction I'll be taking the project.
